### PR TITLE
[Chore] Add k6 testing for the text exercise api

### PR DESCRIPTION
**Pull request by rs-maker**
### Checklist
- [ ] I tested *all* changes and *all* related features with different users (student, tutor, editor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.

### Motivation and Context
 The Text exercise api was the only one not being tested with a corresponding k6 test. 

### Description
 I refactored some code duplication and reproduced the Modeling exercise tests for the text exercise api. 

### Steps for Testing

1. Look at code
2. I opened a temporary bamboo plan to test these changes which you can look at if you have permissions <a href="https://bamboo.ase.in.tum.de/browse/ARTEMIS-ANAT-2">bamboo plan</a> 
3. new tests green
4. old tests also green
Screenshot from 2021-06-19 20-21-16

### CODE REVIEW

**Review by jhainzlschmid — DISMISSED** [review]
Code looks good and the tests run successfully

**Review by FrankeLukas — DISMISSED** [review]

**Review by DanielKainz — DISMISSED** [review]
Code looks good :-)

**Review by iozbeyli — DISMISSED** [review]
Code looks good

**Review by ivohashamov — APPROVED** [review]

**Review by FuchsDominik — APPROVED** [review]
Code looks fine

File: src/test/k6/requests/text.js [hunk 1]
```diff
@@ -21,3 +21,53 @@ export function submitRandomTextAnswerExam(artemis, exercise, submissionId) {
...
+
+export function assessTextSubmission(artemis, exerciseId, resultId) {
+    const assessment = [
+        {
+            credits: 4,
+            reference: 'Package:35f037f7-0606-4798-b43a-9bc76d741421',
+            referenceId: '35f037f7-0606-4798-b43a-9bc76d741421',
+            referenceType: 'Package',
+            text: 'AssessmentText',
+        },
```
Comment (jpbernius) [inline src/test/k6/requests/text.js:62]: Should we use proper Text Feedback here?

**Review by jpbernius — CHANGES_REQUESTED** [review]

File: src/test/k6/requests/text.js [hunk 2]
```diff
@@ -21,3 +21,53 @@ export function submitRandomTextAnswerExam(artemis, exercise, submissionId) {
...
+
+    const res = artemis.post(TEXT_EXERCISES, textExercise);
+    if (res[0].status !== 201) {
+        console.log('ERROR when creating a new text exercise. Response headers:');
+        for (let [key, value] of Object.entries(res[0].headers)) {
+            console.log(`${key}: ${value}`);
+        }
+        fail('FAILTEST: Could not create text exercise (status: ' + res[0].status + ')! response: ' + res[0].body);
+    }
+    console.log('SUCCESS: Generated new text exercise');
```
Comment (iozbeyli) [inline src/test/k6/requests/text.js:49]: Why do you not move this part to exercise file as well?

File: src/test/k6/requests/text.js [hunk 3]
```diff
@@ -21,3 +21,53 @@ export function submitRandomTextAnswerExam(artemis, exercise, submissionId) {
...
+        },
+    ];
+    let res = artemis.put(ASSESS_TEXT_SUBMISSION(exerciseId, resultId), assessment);
+    if (res[0].status !== 200) {
+        console.log('ERROR when assessing modeling (Exercise) via REST. Response headers:');
+        for (let [key, value] of Object.entries(res[0].headers)) {
+            console.log(`${key}: ${value}`);
+        }
+        fail('FAILTEST: Could not assess modeling (Exercise) via REST (status: ' + res[0].status + ')! response: ' + res[0].body);
+    }
```
Comment (iozbeyli) [inline src/test/k6/requests/text.js:71]: This part too

File: src/test/k6/requests/text.js (hunk 1 above)
Comment (rs-maker) [reply src/test/k6/requests/text.js]: We are now using proper text feedback.

**Review by ivanchimeno — APPROVED** [review]
The code looks good

**Review by jpbernius — APPROVED** [review]

**Pull request merged**